mne/source_estimate.py

```diff
@@ -1354,8 +1354,8 @@ class SourceEstimate(_BaseSurfaceSourceEstimate):
         left hemisphere is stored in data[:vertices[0]] and the right
         hemisphere is stored in data[:-vertices[1]].
         When data is a tuple, it contains two arrays:
-            "kernel" shape (n_vertices, n_sensors) and
-            "sens_data" shape (n_sensors, n_times).
+        - "kernel" shape (n_vertices, n_sensors) and
+        - "sens_data" shape (n_sensors, n_times).
```

Comment:
this won't get formatted as bullets without blank lines above and below.  Sorry, should have made that clear in my earlier suggestion.